docs/3.0-migration-guide.d/remove-null-entropy.md

```diff
@@ -0,0 +1,9 @@
+Remove the option to build the library without any entropy sources
+------------------------------------------------------------------
+
+This does not affect users who use the default `config.h`, as this option was
+already off by default.
+
+If you were using the `MBEDTLS_TEST_NULL_ENTROPY` option, you can either use
+`MBEDTLS_ENTROPY_NV_SEED` or create a fake entropy function.
```

Comment:
While technically correct, I'm not sure we should present those two options as equivalent. Suggested wording:

"If you were using the `MBEDTLS_TEST_NULL_ENTROPY` option and your platform doesn't have any entropy source, you should use `MBEDTLS_ENTROPY_NV_SEED` and make sure your device is provisioned with a strong random seed. Alternatively, **for testing purposes only**, you can create and register a fake entropy function."